src/s2/s2cell_id_test.cc

```diff
@@ -367,7 +368,7 @@ TEST(S2CellId, LegacyCoder) {
 
 TEST(S2CellId, LegacyCoderTokenInvalid) {
   // Token too long
-  absl::string_view invalid_token = "000000000000000404\0";
+  string_view invalid_token = "000000000000000404\0";
```

Comment:
Unnecessary \0 at the end?

Reply:
Thanks.  This is pre-existing and will be fixed separately.